### ISSUE TITLE
Add term nasal dermoid cyst

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10346]
**Preferred term label:**
Nasal dermoid cyst

**Synonyms**
Nasal dermoid sinus cyst

**Definition (free text, please give PubMed ID)**
An ectodermal inclusion cyst containing skin and skin appendages; sebaceous glands, hair follicles, and occasionally sweat glands
PMID: 27895540

**Parent term (use hpo.jax.org/app)**
Abnormality of the nose HP:0000366

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM600679

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@added as HP:0430145
